src/standardMethods.ts

```diff
@@ -333,6 +369,9 @@ export function generateSingleReplaceEndpoint<RequestType, RepsonseType>(
       },
       true
     );
-    return metadataMap.addAndReturn(response.data, response);
+    return metadataMap.addAndReturn(
+      response.data,
+      convertAxiosResponseToMetadata(response)
```

Comment:
why do we copy `convertAxiosResponseToMetadata(response)` all the time?
maybe let's just put it inside `addAndReturn`?

Reply:
I will simplify it